portfolio/src/main/webapp/digital.html

```diff
@@ -1,100 +1,118 @@
-    <title>Lucy Qu - Design</title>
-  </head>
 
-  <body>
-    <nav>
-      <div class="logo">
-        <a href="index.html">Lucy Qu</a>
-      </div>
+<head>
+	<link rel="shortcut icon" href="images/logo.png" />
+	<link rel="stylesheet" href="styles.css" />
+	<script type='text/javascript' src="https://www.gstatic.com/charts/loader.js"></script>
+	<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC35R1Q1qkQndHm-Ni6isXLbsSCQif3Umg"></script>
+	<script src="script.js"></script>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+	<title>Lucy Qu - Design</title>
+</head>
 
-      <ul class="nav-links white">
-        <li>
-          <a href="index.html">Projects</a>
-        </li>
-        <li>
-          <a href="other.html">Other</a>
-        </li>
-        <li>
-          <a href="Resume.pdf" target="blank">Resume</a>
-        </li>
-        <li>
-          <a href="about.html">About</a>
-        </li>
-      </ul>
+<body>
+	<nav>
+		<div class="logo">
+			<a href="index.html">Lucy Qu</a>
+		</div>
 
-      <div class="burger">
-        <div class="line1"></div>
-        <div class="line2"></div>
-        <div class="line3"></div>
-      </div>
-    </nav>
+		<ul class="nav-links white">
+			<li>
+				<a href="index.html">Projects</a>
+			</li>
+			<li>
+				<a href="other.html">Other</a>
+			</li>
+			<li>
+				<a href="Resume.pdf" target="blank">Resume</a>
+			</li>
+			<li>
+				<a href="about.html">About</a>
+			</li>
+		</ul>
 
-    <div class="outer-container">
-      <div class="container">
-        <div class="subtitle">
-          Digital
-        </div>
-        <img
+		<div class="burger">
+			<div class="line1"></div>
+			<div class="line2"></div>
+			<div class="line3"></div>
+		</div>
+	</nav>
+
+	<div class="outer-container">
+		<div class="container">
+			<div class="subtitle">
+				Digital
+			</div>
+			<img
           class="large-image"
           src="images/digital/animation.gif"
           alt="chinatown pixel art animation"
         />
-        <br />
-        <img
+			<br />
+			<img
           class="large-image"
           src="images/digital/landscape.jpg"
           alt="landscape vector art"
         />
-        <br />
-        <br />
-        <hr />
-        <div class="subtitle">
-          Japan Infographic
-        </div>
-        <div class="body-text">
-          This project was made for VISA 0150: 2D Foundation. The assignment was
-          to make an infographic, so I chose to make a brochure covering various
-          aspects of Japan such as its history, population, and economy.
-        </div>
-        <br />
+			<br />
+			<br />
+			<hr />
+			<div class="subtitle">
+				Japan Infographic
+			</div>
+			<div class="dropdown-container">
+				<select class="dropdown" onchange="getChart()" name="chart-type" id="chart-type" style="width: 200px">
```

Comment:
When we have an element go over the 80 character limit, we format it with a block indentation so as to help make it more readable
```suggestion
				<select class="dropdown" 
				              onchange="getChart()" 
				              name="chart-type" 
				              id="chart-type" 
				              style="width: 200px">
```